Extensions/ExternalTfs/Src/Tasks/DownloadExternalBuildArtifacts/download.ts

```diff
@@ -73,9 +73,9 @@ async function main(): Promise<void> {
         var parallelLimit: number = +tl.getVariable("release.artifact.download.parallellimit");
 
         var templatePath = path.join(__dirname, 'vsts.handlebars');
-        var buildApi = vssConnection.getBuildApi();
+        var buildApi = await vssConnection.getBuildApi();
 
-        var artifacts = await executeWithRetries("getArtifacts", () => buildApi.getArtifacts(parseInt(buildId), projectId), 3).catch((reason) => {
+        var artifacts = await executeWithRetries("getArtifacts", () => buildApi.getArtifacts(projectId, parseInt(buildId)), 3).catch((reason) => {
```

Comment:
projectid is not required to be passed anymore?

Reply:
We are passing projectId, order has changed.

Reply:
Because of dependency update, older one was deprecated.